frontend/src/scenes/experiments/Metrics/PrimaryGoalTrends.tsx

```diff
@@ -31,70 +35,245 @@ export function PrimaryGoalTrends(): JSX.Element {
+                                                    idx === metricIdx
+                                                        ? { ...metric, exposure_query: undefined }
+                                                        : metric
+                                                ),
+                                            })
+                                        }}
+                                    >
+                                        <div className="font-semibold flex justify-between items-center">
+                                            <span>Default</span>
+                                            {!currentMetric.exposure_query && (
+                                                <IconCheckCircle fontSize={18} color="var(--primary)" />
+                                            )}
+                                        </div>
+                                        <div className="text-muted text-sm leading-relaxed">
+                                            Uses the number of unique users who trigger the{' '}
+                                            <LemonTag>$feature_flag_called</LemonTag> event as your exposure count. This
+                                            is the recommended setting for most experiments, as it accurately tracks
+                                            variant exposure.
+                                        </div>
+                                    </div>
+                                    <div
+                                        className={`flex-1 cursor-pointer p-4 rounded border ${
+                                            currentMetric.exposure_query
+                                                ? 'border-primary bg-primary-highlight'
+                                                : 'border-border'
+                                        }`}
+                                        onClick={() => {
+                                            setExperiment({
+                                                ...experiment,
+                                                metrics: experiment.metrics.map((metric, idx) =>
+                                                    idx === metricIdx
+                                                        ? {
+                                                              ...metric,
+                                                              exposure_query: {
+                                                                  kind: NodeKind.TrendsQuery,
+                                                                  series: [
+                                                                      {
+                                                                          kind: NodeKind.EventsNode,
+                                                                          name: '$pageview',
+                                                                          event: '$pageview',
```

Comment:
What's the rationale for using `$pageview` here?

Reply:
Need to set some default event here, and `$pageview` is our default everywhere. But maybe `$feature_flag_called` makes more sense.